util-core/src/main/java/com/twitter/util/ExceptionalJavaConsumer.java

```diff
@@ -0,0 +1,10 @@
+package com.twitter.util;
+
+/**
+ * Used for Java 8 interop.
+ * This is a SAM version of c.t.u.ExceptionalFunction (returning Unit/void).
+ */
+// @FunctionalInterface
+public interface ExceptionalJavaConsumer<T> {
```

Comment:
is this even worth it given that we can't use FunctionalInterface?  Maybe it would make sense to publish a separate util-lambda target for now?